### ISSUE TITLE
OpenRefine 3.6 creates duplicate Wikidata statements where it is expected to only add qualifiers to existing statements (String-specific?)

### ISSUE THREAD

**Issue by trnstlntk** [OpenRefine/OpenRefine#4728]
In this [Wikidata batch edit](https://editgroups.toolforge.org/b/OR/c00f8770a8e/) ([reverted](https://editgroups.toolforge.org/b/EG/8984072/)), I wanted to add [taxon author](https://www.wikidata.org/wiki/Property:P405) qualifiers to a set of existing [taxon name](https://www.wikidata.org/wiki/Property:P225) Wikidata statements.
But even with the new deduplication settings for Wikidata edits in OpenRefine 3.6, it has created duplicate statements.

I have only done this with [taxon name](https://www.wikidata.org/wiki/Property:P225) statements which supports the String data type. I haven't come across / tested any use case with Item data types yet, and am hence not sure if the problem is specific to statements pointing to non-Item datatypes, or Strings specifically, for instance. (I can imagine it is?)

### Current Results
Duplicate Wikidata statements were created ([example](https://www.wikidata.org/w/index.php?title=Q5063380&oldid=1605615613) / [Edit group](https://editgroups.toolforge.org/b/OR/c00f8770a8e/))

### Expected Behavior
Only a qualifier should be added to the existing statement.

### Versions
 - Operating System: MacOS 10.13.5
 - Browser Version: Chrome 99.0.4844.83
 - JRE or JDK Version: n/a?
 - OpenRefine: 3.6 snapshot release

### Datasets
This one:
(Big messy dataset. Facet or filter the `taxonAuthor-recon` column to only show 'Forel' (reconciled to Auguste Forel), and the `taxonNmCln` column to only reconciled values. That's the set to which I attempted to do a batch edit.)

**Comment by trnstlntk** [comment]
It's been a while since I did this batch, so I don't recall super well, but I do think I used this setting:
![image](https://user-images.githubusercontent.com/9881645/162612760-1502fba1-c706-4348-bdd5-b3005949159b.png)

I have just tried again with the 'property and value' matching strategy, but that one seems to auto-jump to 'property' only when I re-open the configuration dialog.
![image](https://user-images.githubusercontent.com/9881645/162612796-dd5dbe45-e4a7-4cc4-a5eb-1a759ad40c82.png)
When re-opening this dialog, it has jumped to this:
![image](https://user-images.githubusercontent.com/9881645/162612809-f275358a-91fb-4b14-a47e-458446cfe826.png)

This setting does not produce any Wikidata edits, although I do see a preview:
![image](https://user-images.githubusercontent.com/9881645/162612900-00ff6d00-2ac5-49be-815b-6477b0bbac02.png)

By the way, because of ##4727 I am still using an early March 2022 (I think March 10) snapshot release, not a recent one.

**Comment by thadguidry** [comment]
@trnstlntk Is this still an issue with the latest snapshot?  If yes, that's fine, we'll keep this bug issue open.

**Comment by trnstlntk** [comment]
Yes, I tried with the same project and got duplicate statements again. I then switched to other matching strategies but those didn't trigger edits at all. Same behavior as I described in the comments above. I think this still needs investigation, let's keep the issue open.

Tested with Version 3.7-SNAPSHOT [9948f8d] and with the same dataset I linked above.
